client_encryption/field_level_encryption.py

```diff
@@ -47,7 +50,10 @@ def decrypt_payload(payload, config, _params=None):
     """Decrypt some fields of a JSON payload using the given configuration."""
 
     try:
-        json_payload = payload if type(payload) is dict else json.loads(payload)
+        if type(payload) is dict or type(payload) is list:
```

Comment:
For adderessing: https://sonarcloud.io/project/issues?resolved=false&types=CODE_SMELL&branch=fix_array_payload&id=Mastercard_client-encryption-python&open=AYMOH17JUj1Lk7_J0HyC can use 

```
json_payload = payload if type(payload) is dict or type(payload) is list else json.loads(payload)
```